org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -27,6 +27,9 @@ <h3>New Features</h3>
   <li>Methods <code>toString</code>, <code>hashCode</code> and <code>equals</code>
       generated by compiler for records are filtered out during generation of report
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/990">#990</a>).</li>
+  <li>Bridge methods not marked as <code>synthetic</code> by Kotlin compiler
```

Comment:
Even if we discovered this because of the Kotlin compiler we filter bridge methods in any case (regardless of the compiler). Maybe we just say: _Bridge methods are filtered out during report generation_?

Reply:
@marchof done